pkg/cache/informer.go

```diff
@@ -197,12 +200,16 @@ func (in *DefinitionCache) Start(ctx context.Context, store cache.Cache, duratio
 	})
 	if err != nil {
 		klog.ErrorS(err, "failed to add event handler for definition cache")
+		return
```

Comment:
do we really need to return here? ping @Somefive

Reply:
In fact, the error is only returned after the informer is stopped, so it can be returned directly here